data/models/projects/Fun/info.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Fun",
```

Don't you wanna give more context? Maybe:
```suggestion
  "name": "Fun programming language",
```

---

Yeah, this is good